File: test/fizzbuzz/fizzbuzz.java
public class FizzBuzz {
	public static void main(String[] args) {
		for(int i = 1; i <= 100; i++) {
			if (((i % 5) == 0) && ((i % 7) == 0))
				System.out.print("fizzbuzz");    
			else if ((i % 5) == 0) System.out.print("fizz");
			else if ((i % 7) == 0) System.out.print("buzz");
			else System.out.print(i);
			System.out.print(" "); 
		}
		System.out.println();
	}
}

File: test/fizzbuzz/fizzbuzz.rb
1.step(100,1) do |i|
    if (i % 5) == 0 && (i % 3) ==0
        puts 'FizzBuzz'
    elsif (i % 5) == 0
        puts 'Buzz'
    elsif (i % 3) == 0
        puts 'Fizz'
    else
        puts i
    end
end